account_invoice_margin_sale_delivered_sync/models/res_company.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2022 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    margin_sale_sync_invoice_posted = fields.Boolean(
```

Comment:
company_dependent=True

Reply:
The field is in the company, so there's no sense to make it company dependent

Reply:
you are right